project2/Navigation.js

```diff
@@ -1,16 +1,29 @@
 import { createAppContainer } from "react-navigation";
-import { createStackNavigator } from 'react-navigation-stack';
+import { createStackNavigator } from "react-navigation-stack";
 
-import searchScreen from "./screens/search";
-import movieScreen from "./screens/movie";
+import SearchScreen from "./screens/Search";
+import MovieScreen from "./screens/Movie";
 
 const stackNavigator = createStackNavigator(
-    {
-        "Search": searchScreen,
-        "Movie": movieScreen,
+  {
+    Search: {
+      screen: SearchScreen,
+      navigationOptions: {
+        title: "Search",
+      },
     },
-    {
-        initialRouteName: "Search",
-    }
-)
-export default createAppContainer(stackNavigator);
\ No newline at end of file
+    Movie: {
+      screen: MovieScreen,
+      navigationOptions: {
+        title: "Movie Details",
+      },
+    },
+  },
+  {
+    defaultNavigationOptions: {
+      headerTitleStyle: { color: "#AAAAAF" },
+    },
```

Comment:
Any navigation functionality common to all items in a stack can be put under `defaultNavigationOptions`, in this case the colour.